src/main/java/com/home/ROMS/service/user/UserRepository.java

```diff
@@ -3,6 +3,6 @@
 import com.home.ROMS.domain.user.User;
 import org.springframework.data.jpa.repository.JpaRepository;
 
-interface UserRepository extends JpaRepository<User, Long> {
+public interface UserRepository extends JpaRepository<User, Long> {
```

Comment:
sterge "public". Interfata trebuie accesata doar prin intermediul UserService (faci o metoda in UserService care apeleaza mai departe metoda din UserRepository)